localization/languages/pt-BR.json

```diff
@@ -35,29 +35,29 @@
         "viewSettings": "Ver configurações",
         "viewReadingList": "Ver listas de leitura",
         "takeScreenshot": "Capturar tela",
-        "clearHistory": "Limpar todo o histórico",
+        "clearHistory": "Limpar histórico",
         "clearHistoryConfirmation": null, //missing translation
         "switchToTask": "Trocar para a tarefa",
-        "createTask": "Criar uma tarefa",
-        "closeTask": null, //missing translation
+        "createTask": "Criar tarefa",
+        "closeTask": "Fechar tarefa",
         "moveToTask": "Mover esta aba para uma tarefa",
-        "nameTask": null, //missing translation
-        "addBookmark": null, //missing translation
+        "nameTask": " Nomear tarefa",
+        "addBookmark": "Adicionar favorito",
 		"searchBookmarks": "Buscar nos favoritos",
         "bookmarksAddTag": null, //missing translation
 		"bookmarksSimilarItems": null, //missing translation
-		"searchHistory": null, //missing translation
-        "importBookmarks": null, //missing translation
-		"exportBookmarks": null, //missing translation
+		"searchHistory": "Buscar no histórico",
+        "importBookmarks": "Importar favoritos",
+		"exportBookmarks": "Exportar favoritos",
         /* navbar */
-        "openMenu": null, //missing translation
-        "enterReaderView": "Entrar no modo leitura",
+        "openMenu": "Menu",
+        "enterReaderView": "Modo leitura",
         "exitReaderView": "Sair do modo leitura",
         "emptyReadingListTitle": "Sua lista de leitura está vazia",
         "emptyReadingListSubtitle": "Artigos abertos no modo de leitura são listados aqui, e salvos offline por 30 dias.",
         "newTabLabel": "Nova aba", //this is a noun, used for tabs that don't have a page loaded in them yet
         "connectionNotSecure": "Sua conexão com este site não é segura.",
-        "searchbarPlaceholder": "Busque ou digite um endereço",
+        "searchbarPlaceholder": "Pesquisar...",
```

Comment:
Can you please make the translation match the [English version](https://github.com/minbrowser/min/blob/master/localization/languages/en-US.json#L60) (as well as for the other translations)?